addons/advanced_throwing/functions/fnc_prime.sqf

```diff
@@ -44,11 +44,11 @@ if (_ammoCount == 1) then {
 _unit setAmmo [_muzzle, 0];
 
 private _throwableType = getText (_config >> "ammo");
+private _ammoConfig = configFile >> "CfgAmmo" >> _throwableType;
 
 // Handle weird scripted grenades (RHS) which could cause unexpected behaviour
-private _nonInheritedCfg = configProperties [configFile >> "CfgAmmo" >> _throwableType, 'configName _x == QGVAR(replaceWith)', false];
-if ((count _nonInheritedCfg) == 1) then {
-    _throwableType = getText (_nonInheritedCfg select 0);
+if (inheritsFrom (_ammoConfig >> QGVAR(replaceWith)) isEqualTo _ammoConfig) then {
```

Comment:
Fyi `QGVAR(replaceWith)` isn't actually used anywhere, but I'm keeping it for BWC, despite it not being documented.